patches/tModLoader/Terraria/Social/Steam/WorkshopHelper.TML.cs

```diff
@@ -72,9 +72,6 @@ internal static void PublishMod(LocalMod mod, string iconPath)
 		var modFile = mod.modFile;
 		var bp = mod.properties;
 
-		if (bp.buildVersion != modFile.TModLoaderVersion)
-			throw new WebException(Language.GetTextValue("OutdatedModCantPublishError.BetaModCantPublishError"));
-
```

Comment:
Why is this removed?
AFAIK, this was a 1.3 Mod Browser check to ensure the file is self-coherent.